lib/recipe/widgets/probe_form.dart

```diff
@@ -17,38 +17,46 @@ class ProbeForm extends StatelessWidget {
-                    helperText:
-                        'Recipe probe - e.g. Which flour did you use to make roti?',
-                    errorText: (state.recipes[recipeIndex].probes[probeIndex]
-                                    .probeName !=
-                                null &&
-                            state.recipes[recipeIndex].probes[probeIndex]
-                                .probeName!.isEmpty)
-                        ? 'Probes cannot be empty'
-                        : null),
-                onChanged: (value) {
-                  context.read<RecipeBloc>().add(ProbeChanged(
-                      recipe: state.recipes[recipeIndex],
-                      probeName: value,
-                      probeIndex: probeIndex));
-                },
-                textCapitalization: TextCapitalization.sentences,
-                textInputAction: TextInputAction.next,
+              AbsorbPointer(
+                absorbing: state.recipes[recipeIndex].saved,
+                child: Column(
+                  children: [
+                    TextFormField(
+                      initialValue:
+                          state.recipes[recipeIndex].probes[probeIndex].name,
+                      decoration: InputDecoration(
+                          icon: const Icon(Icons.live_help),
+                          labelText: 'Probe ${probeIndex + 1}',
+                          helperText:
+                              'Recipe probe - e.g. Which flour did you use to make roti?',
+                          errorText: (isFieldModifiedAndEmpty(state
+                                  .recipes[recipeIndex]
+                                  .probes[probeIndex]
+                                  .name))
+                              ? 'Probes cannot be empty'
+                              : null),
+                      onChanged: (value) {
+                        context.read<RecipeBloc>().add(ProbeChanged(
+                            recipe: state.recipes[recipeIndex],
+                            probeName: value,
+                            probeIndex: probeIndex));
+                      },
+                      textCapitalization: TextCapitalization.sentences,
+                      textInputAction: TextInputAction.next,
+                    ),
+                    const SizedBox(height: 10),
+                    const ListTile(title: Text('Probe options:')),
+                  ],
+                ),
               ),
-              const ListTile(title: Text('Edit probe options:')),
               Expanded(
                 child: ListView.builder(
                     padding: const EdgeInsets.all(2.0),
                     itemCount: state.recipes[recipeIndex].probes[probeIndex]
                         .probeOptions.length,
                     itemBuilder: (context, index) {
                       return Slidable(
+                          enabled: !state.recipes[recipeIndex].saved,
```

Comment:
Don't we want to get rid of slidable altogether now that major UI interactions have been changed to long press?

Reply:
Hadn't changed this since I thought we still had the issue with long press on textfields, hence that's the only places left with slidable

Reply:
Ahh, yeah, I remember. Yup, we can have a look into that. I remember I've linked the issue from the main flutter repo somewhere, so we can have a look if that has been merged to the stable branch. There's a new flutter version out, so maybe upgrading Gibsonify to it would enable us to use long-presses on textfields.